0008-interledger-quoting-protocol/0008-interledger-quoting-protocol.md

```diff
@@ -33,108 +31,90 @@ The sending and receiving ledger are different ledgers. Otherwise, there should
 
 ## Get Quote
 
-Quotes are requested with `plugin.sendMessage(message)` and returned via `plugin.on("incoming_message")` (see the Ledger Plugin Interface for more details).
-The message formats described here are the `data` properties of an `OutgoingMessage`.
+Quotes are requested with `plugin.sendRequest(message)` and returned via `plugin.registerRequestHandler(fn)` (see the Ledger Plugin Interface for more details).
 
-### Request Message Format
+A quote request message's `ilp` property must be an `IlqpLiquidityRequest`, `IlqpBySourceRequest`, or `IlqpByDestinationRequest`. The response message's `ilp` must be an `IlqpLiquidityResponse`, `IlqpBySourceResponse`, or `IlqpByDestinationResponse` respectively. If an error occurs during quoting, `ilp` will be a [`IlpError`](../0003-interledger-protocol/0003-interledger-protocol.md#ilp-error-format) instead.
 
-| Field    | Type            | Description |
-|:---------|:----------------|:------------|
-| `method` | String          | `"quote_request"` |
-| `id`     | String          | A UUID to match this request to the corresponding response. |
-| `data`   | QuoteParameters | An Object describing the quote parameters. |
+## ILQP Packets
+### IlqpLiquidityRequest
 
-##### QuoteParameters
+| Field | Type | Short Description |
+|:--|:--|:--|
+| destinationAccount | Address | Address corresponding to the destination account |
```

Comment:
If you request a `destinationPrefix`, the connector could reply that there is no single curve that corresponds to the entire curve.

Instead, requesting a `destinationAccount` means that the connector can select to appropriate `appliesToPrefix`.